### ISSUE TITLE
WatsonApiException: Error: Unauthorized: Access is denied due to invalid credentials , Code: 401 , X-dp-watson-tran-id: gateway01-988173465

### ISSUE THREAD

**Issue by RVKRM** [watson-developer-cloud/python-sdk#454]
Am getting Watson api exception error. Am using python 3.5/ Jupyter lab. i have doubled checked the credentials its correct
```
ctx = {
  "url": "---",
  "username": "---",
  "password": "---"
}

workspace_id = '---'

language = 'en'

version = '2018-02-16'

```

```
def load_intents(export=False):
    response = {'pagination': { 'next_cursor': ''}}

    questions = []
    while 'next_cursor' in response['pagination']:
        cursor = response['pagination']['next_cursor']
        response = conversation.list_intents(workspace_id=workspace_id,export=export, cursor=cursor)

        for intent in response['intents']:
            for example in intent['examples']:
                row = { 
                    'Question': example['text'],
                    'Intent': intent['intent']
                }
                questions.append(row)

    df = pd.DataFrame(questions, columns=['Question','Intent'])
    
    return df

df = load_intents(export=True)
print('Number of records: {}'.format(len(df)))
df.head()
```

**Comment by germanattanasio** [comment]
Can you provide clear steps for us to reproduce the issue? It's very hard to follow your code. Please remove all the code that is not related to this issue.

**Comment by germanattanasio** [comment]
@RVKRM I can find what the problem is from the stacktrace. What I need is something like the steps below:

## Steps to reproduce the problem

1. Install version X of the SDK
1. Create Python file with this content
    ```py
    # some code here that can be run with `python filename.py`
    ```
1. List all the external dependencies like APIs, files that the example code needs.

**Comment by germanattanasio** [comment]
This issue is fairly old and there hasn't been much activity on it. Closing, but please re-open if it still occurs.
